mono/mini/debugger-agent.c

```diff
@@ -4516,7 +4772,11 @@ clear_breakpoint (MonoBreakpoint *bp)
 	for (i = 0; i < bp->children->len; ++i) {
 		BreakpointInstance *inst = (BreakpointInstance *)g_ptr_array_index (bp->children, i);
 
-		remove_breakpoint (inst);
+#ifndef IL2CPP_MONO_DEBUGGER
```

Comment:
Just a general comment. I'd prefer this macro not be checked in a negative sense.... if not il2cpp mono debugger... a bit confusing, and I've misread this many times. I'd prefer something like #ifdef IL2CPP_MONO_DEBUGGER.. or better yet just IL2CPP_DEBUGGER

Reply:
Good point. I've add this to our Trello card about cleaning up the defines. I think we should pick either the positive or negative sense and stick with it.